tests/test_iss_data.py

```diff
@@ -7,6 +7,8 @@
 import tempfile
 import unittest
 
+import jsonpath_rw
```

Comment:
Is this sophisticated enough to warrant inclusion as a dependency? 

Why can't you just: 
```
with open(json_filepath, "r") as fh:
    document = json.load(fh)
return document['codebook'][0]
```

 Perhaps I don't understand what is happening in this function.